integration

```diff
@@ -618,6 +618,7 @@
   "plamish/xcomfort",
   "plmilord/Hass.io-custom-component-spaclient",
   "Poeschl/Remote-PicoTTS",
+  "popeen/Home-Assistant-Custom-Component-MotalaVattenAvfall",
```

Comment:
This looks like a Swedish service, if it is, you should add `country` key in hacs.json

Reply:
Yea, its Swedish.
Just added the country key